### PR TITLE
DOCSP-35980: Insert One usage example

### PR DESCRIPTION
**Pull request by norareidy**
Adds a usage example showing how to insert a document.
JIRA - https://jira.mongodb.org/browse/DOCSP-35980
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/laravel/DOCSP-35980-staging/usage-examples/insertOne/

### Checklist

- [ ] Add tests and ensure they pass
- [ ] Add an entry to the CHANGELOG.md file
- [ ] Update documentation for new features

### CODE REVIEW

File: docs/includes/usage-examples/InsertOneTest.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        require_once __DIR__ . '/Movie.php';
+
+        Movie::truncate();
+
+        // begin-insert-one
+        $movie = Movie::create([
+            'title' => 'Marriage Story',
+            'year' => 2019,
+            'runtime' => 136,
+        ]);
```
Comment (GromNaN) [inline docs/includes/usage-examples/InsertOneTest.php:27]: This example could be completed by the more expressive form (if anyone need to update the `$movie` model before saving):

```php
$movie = new Movie([
    'title' => 'Marriage Story',
    'year' => 2019,
    'runtime' => 136,
]);
$movie->save();
```

File: docs/usage-examples/insertOne.txt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+   :keywords: insert one, add one, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can insert a document into a collection by calling the ``create()`` method on
+an Eloquent model or on a query builder.
```
Comment (rustagir) [inline docs/usage-examples/insertOne.txt]:
```suggestion
You can insert a document into a collection by calling the ``create()`` method on
an Eloquent model or query builder.
```

File: docs/usage-examples/insertOne.txt [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          "title": "Marriage Story",
+          "year": 2019,
+          "runtime": 136,
+          "updated_at": "...",
+          "created_at": "...",
+          "_id": "..."
+      }
+
+For instructions on editing your Laravel application to run the usage example, see the
+:ref:`Usage Example landing page <laravel-usage-examples>`.
```
Comment (rustagir) [inline docs/usage-examples/insertOne.txt]:
```suggestion
To learn how to edit your Laravel application to run the usage example, see the
:ref:`Usage Example landing page <laravel-usage-examples>`.
```

File: docs/usage-examples/insertOne.txt [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+.. tip::
+
+   You can also use the ``save()`` or ``insert()`` methods to insert a document into a collection.
+   To learn more  about insert operations, see the :ref:`laravel-fundamentals-insert-documents` section
```
Comment (rustagir) [inline docs/usage-examples/insertOne.txt:70]: Q: will this be merged only after write operations is merged? Link is broken as of now

**Review by rustagir — APPROVED** [review]
lgtm with a few comments!

File: docs/usage-examples/insertOne.txt (hunk 4 above)
Comment (norareidy) [reply docs/usage-examples/insertOne.txt:70]: Yeah I'll wait until after write operations is merged

File: docs/usage-examples/insertOne.txt [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can insert a document into a collection by calling the ``create()`` method on
+an Eloquent model or query builder.
+
+To insert a document, call the ``create()`` method and specify the document's field values
+inside the method call.
```
Comment (GromNaN) [inline docs/usage-examples/insertOne.txt]: It seems to me that this 2 sentences tell the same thing. 
```suggestion
You can insert a document into a collection by calling the ``create()`` method on
an Eloquent model, specifying the values for each field.
```

File: docs/usage-examples/insertOne.txt [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+   :copyable: true
+
+   .. input:: ../includes/usage-examples/InsertOneTest.php
+      :start-after: begin-insert-one
+      :end-before: end-insert-one
+      :language: php
+      :dedent:
+
+   .. output::
+      :language: console
```
Comment (GromNaN) [inline docs/usage-examples/insertOne.txt]: This is valid JSON. Can you format with a language: "json", "js" or "ts"?

File: docs/includes/usage-examples/InsertOneTest.php (hunk 1 above)
Comment (ccho-mongodb) [reply docs/includes/usage-examples/InsertOneTest.php:27]: @GromNaN Would you recommend replacing the ``create()`` example with one that shows ``save()``? If ``save()`` seems more common and helpful, we could update the example to show that method of inserting a single document instead.

Comment (GromNaN) [reply docs/includes/usage-examples/InsertOneTest.php:27]: No, `Movie::create([...])` is the most common way to create a model.

There is also `Movie::createOrFirst()` that can be used (using `findOneAndUpdate` with upsert)

**Review by GromNaN — APPROVED** [review]
LGTM as this.

**Pull request merged**